kbdd/src/test/kotlin/ru/fix/kbdd/rest/RestTest.kt

```diff
@@ -392,4 +393,31 @@ class RestTest {
                             """))
         )
     }
+
+    @Test
+    suspend fun `post multipart data`() {
+        request {
+            headers { "my-header" % "header-value" }
+            baseUri(server.baseUrl())
+            post("/post-form-data-request")
+            file("resource-with-placeholders.json", File("src/test/resources/resource-with-placeholders.json").inputStream())
```

Comment:
File tries to open resource based on current working directory. Better to use 
`RestTest::class.java.getResourceAsStream("/resource-with-placeholders.json")` this way we are  looking in test resource independently on build environment settings.